src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -67,106 +67,118 @@
 {% macro caseStatusForClaimant(status, claim) %}
   {% if status === ClaimStatus.NO_RESPONSE %}
     {{ noResponseForClaimantDashboard() }}
-  {% elif status === ClaimStatus.MORE_TIME_REQUESTED %}
+    {% elif status === ClaimStatus.MORE_TIME_REQUESTED %}
```

Comment:
Try to avoid unnecessary changes - they pollute the PR. Ideally I'd like to see these indents reverted. (Seems that the auto-formatter doesn't understand `elif`.)

Reply:
True, I think its the auto formatter